app/_kong_plugins/request-termination/index.md

```diff
@@ -31,6 +30,37 @@ categories:
 
 search_aliases:
   - request-termination
+
+related_resources:
+  - text: Allow clients to choose their authentication methods and prevent unauthorized access
+    url: /how-to/allow-multiple-authentication/
 ---
 
-## Overview
+This plugin terminates incoming requests with a specified status code and message.
+This can be used to temporarily stop traffic on a Gateway Service or a Route, or even block a Consumer. 
```

Comment:
```suggestion
This can be used to temporarily stop traffic on a [Gateway Service](/gateway/entities/service/) or a [Route](/gateway/entities/route/), or even block a [Consumer](/gateway/entities/consumer/). 
```